collect_app/src/main/java/org/odk/collect/android/activities/GeoPointActivity.java

```diff
@@ -161,6 +189,14 @@ private void setupLocationDialog() {
                 "show");
         // dialog displayed while fetching gps location
         locationDialog = new ProgressDialog(this);
+
+        // back button doesn't cancel
+        locationDialog.setCancelable(false);
```

Comment:
I'm curious the reason why this is not cancelable

Reply:
I was thinking about that too, don't know maybe to avoid canceling accidentally, you know sometimes users wait maybe even a few minutes to catch a good location so it would be annoying to cancel the process by accident.

Reply:
If it was just the back button would cancel, I'd be OK with that because GPS is resumable. I talked to @lognaturel and she said the compelling reason is that if you tap outside the dialog, the dialog will cancel. I think that's disconcerting to users. 

Let's keep cancelable set to false, but please change your comment to mention the above compelling reason.

Reply:
Fixed.